karma.conf.js

```diff
@@ -88,6 +88,8 @@ const babelOptions = (options = {}) => {
 	};
 };
 
+sauceLabs = true;
```

Comment:
what's this for?

Reply:
Make sauceLabs run on these tests so I can see if the IE11  tests are going well, hence the `(remove this when verified)` in said commit

Reply:
ah, missed the commit. Seems like our travis setup needs to be tweaked - we should be running saucelabs for branches in the main trepo, just not forks.